.vib/prestashop/cypress/cypress.json

```diff
@@ -0,0 +1,14 @@
+{
+    "env": {
+      "username": "user",
+      "email": "user@example.com",
+      "password": "ComplicatedPassword123!4"
+    },
+    "hosts": {
+      "vmware-prestashop.my": "{{ TARGET_IP }}"
+    },
+    "defaultCommandTimeout": 30000,
+    "viewportWidth": 1800,
```

Comment:
Why do we need to adjust the port height? Just curious.

Reply:
Also, I think you need to pass this file through linter.

Reply:
The dimensions should be adjusted in order to prevent the mobile view of the pages to kick in.